spec/integration/node/support/keep-alive.js

```diff
@@ -0,0 +1,21 @@
+import createManifest from 'spec/integration/support/manifest'
+
+import forge, { configs } from 'src/index'
```

Comment:
test locally that `yarn test` runs 

```suggestion
import forge from 'src/index'
```